src/main/java/mate/academy/internetshop/controller/orderscontroller/CreateOrderController.java

```diff
@@ -0,0 +1,33 @@
+package mate.academy.internetshop.controller.orderscontroller;
```

Comment:
```suggestion
package mate.academy.internetshop.controller.order;
```
аналогічно інші, якщо є